addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/internal/serializer/CustomTankerkoenigDetailResultDeserializer.java

```diff
@@ -43,13 +43,15 @@ public TankerkoenigDetailResult deserialize(final JsonElement json, final Type t
             final Double e10 = jsonStation.get("e10").getAsDouble();
             final Double e5 = jsonStation.get("e5").getAsDouble();
             final Double diesel = jsonStation.get("diesel").getAsDouble();
+            final Boolean isOpen = jsonStation.get("isOpen").getAsBoolean();
```

Comment:
Does it return a `boolean` or `Boolean`, the difference is that the last one supports null, but if you pass it to a method which only accepts boolean, it will throw an NPE if its actually null. If this line can return null maybe you should change it to accepts a `Boolean` instead.

I doubt whether I completely understand it, does it set open based on the status (see WebserviceHandler) or does the webservice deliver it via the field isOpen?